### DISCUSSION TITLE
🌍 The "Blue NFT" : A NFT for the planet

### DISCUSSION THREAD

**Discussion by jeemclr** [capsule-corp-ternoa/ternoa-proposals#45] [Ideas]
Blockchain is often mentioned as a not sustainable technology. Even if all the blockchain space is working to make it more and more "green" everyday, for the general public it's not something palpable.

My idea there, is to share thinking about how we could create a NFT dedicated for the planet. It can take lot of forms. Reduce the energy consumption of NFTs, create a treasury in order to use the funds for the planet, create a special NFT which aims to protect the resources of the planet, etc.

Open the discussion and share our ideas, our thoughts to help the Earth, our Blue Planet, keep its virtues and its beauty in order to prosper through the ages.

**Comment by Orius** [comment]
Hello.
I suggest you to get closer to the Monopole project, which shares the same values.
They work with the Ternoa Labs.
More than greenwashing, they propose to support all blockchain projects that go in this direction (as an incubator), and they could help you. Don't hesitate to share your ideas with their team, which is very attentive.
https://monopole.network/

**Reply by jeemclr** [reply]
Thanks @Orius. The purpose there is to create on-chain solution that could be used by all projects they want participate to "save the planet". Monopole looks great and goes to this direction. It can be really interesting to have your insights on how we can improve or create a specific feature that help such a project like this to fit with your value proposition.

**Comment by markopoloparadox** [comment]
I think this is really a great idea. We already have spend some time brainstorming on how we can efficiently use the Treasury tokens and this could be our first real use case.

What do you think @Orius, would it make sense to create a treasury proposal which could be used to support one of the save-the-world causes?  (this is just an example)

**Reply by Orius** [reply]
Hello Marko.
I think there is no point in reinventing what already exists. It is the essence of Monopole to support eco-responsible projects, respectful of the Earth and the Human being.
The Monopole foundation, based in Switzerland, aims at decarbonation, so that the project as a whole has a negative carbon balance.
All projects supported by Monopole, whether cultural, real estate, crypto ... must strictly comply with the United Nations specifications in terms of respect for the environment and for a sustainable economy.
As you probably know, the $MNPL token will be cross-chain, and it will use the Ternoa blockchain from the launch. By the way, Monopole is supported by the Ternoa Labs.
Without giving up an investment through Ternoa's treasury, I think the best thing would have been to support Monopole through the Ternoa Launchpad. In this respect, the file was closed and most of the Ternoa Team supported the Launchpad. Nobody understood Mika's veto (especially within Team Monopole).
I think that if there is no Launchpad (if it is not recoverable), an investment by the Ternoa treasury will have to be dealt with directly between the top managers (Justine Houge at Monopole, Clément Tequi has her contact information).
Personally, I am only the Communauty Manager and system and network administrator of Orius International, so I cannot make any proposal, only give my opinion.

**Reply by markopoloparadox** [reply]
For some reason when I read you original message I thought that by "they work with Ternoa Labs" you meant that some activities are going on and not that a fully fledged partnership was established. I should probably work on my reading comprehension :laughing: 

I am quite cautious regarding any project that wants to save the world in one way or the other way (a lot of them are legal scams) so I though that it could be interesting to use the blockchain tokens to fund many different green web2 and web3 initiatives. This doesn't mean that I don't trust Monople it just a matter of being pragmatical.
